src/CLI.hs

```diff
@@ -185,24 +190,32 @@ prove file' solver' smttimeout' debug' = do
 coq' :: FilePath -> IO ()
 coq' f = do
   contents <- readFile f
-  proceed contents (enrich <$> compile contents) $ \claims ->
+  proceed contents (enrich <$> compile contents) $ \claims -> do
+    checkCases claims
     TIO.putStr $ coq claims
 
 
+-- checkOverlaps :: FilePath -> IO ()
```

Comment:
This is commented out because it give a warning since it is unused. But it is nice to have a top-level function to check only for case consistency (at least it is helpful for debugging)

Reply:
maybe it can go in `Dev.hs`?